contrib/systemd/auto-update/podman-auto-update.service

```diff
@@ -6,8 +6,8 @@ After=network-online.target
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/podman auto-update
-ExecStartPost=/usr/bin/podman image prune -f
+ExecStart=@@PODMAN@@ auto-update
```

Comment:
This will break anyone who uses these files directly.

Reply:
Does that actually happen? (Serious question. I really don't know).

Reply:
Please fight this out with @edsantiago and @cevich.

Reply:
> Does that actually happen? (Serious question. I really don't know).

I do not know some. Packages should use `make install`.

Reply:
I am not sure about this, what about debian? @siretart Could you confirm if you use the files directly without the makefile target?

Reply:
I think direct-use of the files should be discouraged.  This can be done explicitly by giving them a `.in` or `.template` extension.

Reply:
@Luap99 thanks for reaching out. Indeed, debian uses the top-level Makefile only for building documentation and manpages. The compilation of the golang parts is done using a very bespoke infrastructure (called golang-debhelper). The installation of the the systemd service files, etc., is done manually.

If you require a generation step for the service, I'd appreciate if it wouldn't edit the file inplace (so I really like the `.in` suffix idea), and have an explicit target in the `Makefile` that I could call during the package build.